hg-fast-export.py

```diff
@@ -375,19 +376,19 @@ def get_branchname(name):
   return checkpoint(count)
 
 def export_note(ui,repo,revision,count,authors,encoding,is_first):
-  (revnode,_,user,(time,timezone),_,_,_,_)=get_changeset(ui,repo,revision,authors,encoding)
-  if repo[revnode].hidden():
-    return count
+  ctx = repo[revision]
 
-  parents = [p for p in repo.changelog.parentrevs(revision) if p >= 0]
+  (_,user,(time,timezone),_,_,_,_)=get_changeset(ui,repo,revision,authors,encoding)
+  if ctx.hidden():
```

Comment:
I just noticed this `ctx.hidden()` can also be done before `get_changeset()` now.

Reply:
Feel free to ammend this commit or add a new one.